RELEASE.md

```diff
@@ -1,5 +1,10 @@
 # RELEASE NOTES
 
+## v3.0.6 - Powerflow Animation Update
+
+* Update to latest pypowerwall proxy t29:
+* Default page loaded by proxy (http://localhost:8675/) will render Powerflow Animation. Animation assets (html, css, js, images, fonts, svg) will render from local filesystem instead of pulling from Powerwall TEG portal resulting in faster render time.
```

Comment:
Committed a change here to the proxy address to "http://localhost:8675/" be consistent with links in the README for Powerwall-Dashboard.

Reply:
Perfect! Thanks @mcbirse .